### PR TITLE
Added Jokes Program using API

### PR DESCRIPTION
**Pull request by imsuraj675**
Closes: #4 

- **Title** :  Added Jokes Program using API
- **Name:**: Suraj Kashyap
- **Idenitfy yourself:** SSOC

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
I have added the required Jokes program under API programs section

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/Contributing.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: src/apps/pages/programs/ApiPrograms/Jokes.py
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/ApiPrograms/Jokes.py:1]: filename start with small letters. so change the name

File: src/apps/pages/programs/apiProgram.py [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 def apiPrograms():
   st.title('API Programs')
-  choice = st.selectbox('Select a program to execute', [None])
-
+  choice = st.selectbox('Select a program to execute', [None, "Jokes"])
+  if choice == "Jokes":
+    from src.apps.pages.programs.ApiPrograms.Jokes import play_joke
+    play_joke()
   st.markdown('---')
```
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/apiProgram.py:9]: `st.markdown('---')` keep it above if statement

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@imsuraj675 attach a working video.

Is this api is free of cost?

File: src/apps/pages/programs/ApiPrograms/Jokes.py
Comment (imsuraj675) [reply src/apps/pages/programs/ApiPrograms/Jokes.py:1]: https://github.com/Avdhesh-Varshney/Jarvis/assets/120774752/ba5b1630-d14d-4268-bb0e-34f2da710b71

This is the testing video, I have used API using this repository: https://github.com/15Dkatz/official_joke_api

**Review by Avdhesh-Varshney — APPROVED** [review]
@imsuraj675 PR Approved 🎉

**Comment by imsuraj675** [comment]
In this issue, earlier I was trying to add audio effects, but I was unable to do so, when you have set the priority to high, I was forced to submit the work whatever I had done. But now, I found the solution to add sound effects. Here is the video showing the changes, which I want to do. If permitted, will I have to open a new issue or directly raise a new PR indicating this PR.

Till now I have added 10 random laughing/funny sound. The sounds are present inside the assets folder inside a Jokes_Sound directory. All have a duration of around 3-5sec.

https://github.com/Avdhesh-Varshney/Jarvis/assets/120774752/af5dc789-b935-470a-afc7-7d381ddf6945

**Comment by Avdhesh-Varshney** [comment]
@imsuraj675 idea of sounds looks nice. But these instruments sounds are not looks good for jokes feature. If you want to add, search for something laughing sounds, so that user will motivate to laugh, whatever be the jokes.

`assets/sounds/...` ok

**Pull request merged**